src/main/java/com/google/devtools/build/lib/actions/ResourceSet.java

```diff
@@ -184,13 +120,7 @@ public int getLocalTestCount() {
   @Override
   public String toString() {
     return "Resources: \n"
-        + "Memory: "
-        + memoryMb
-        + "M\n"
-        + "CPU: "
-        + cpuUsage
-        + "\n"
-        + Joiner.on("\n").withKeyValueSeparator(": ").join(extraResourceUsage.entrySet())
+        + Joiner.on("\n").withKeyValueSeparator(": ").join(resources.entrySet())
```

Comment:
Can you preserve the behavior of this? I think it'd be better to show Memory and CPU at the top since they are the more typical resources, then the rest of the resources below.

Reply:
Done